builds/misc/mqtt-perf.yaml

```diff
@@ -0,0 +1,78 @@
+# MQTT Performance tests pipeline
+# Each platform separated into stages
+# Each stage consists of 
+# - setup job for test host (pull and start 'mqttd' container). See templates/mqtt-perf-setup.yaml
+# - setup job for publisher node and subscriber node (mqtt-benchmark tool writen in go). See templates/mqtt-perf-setup.yaml
+# - a set of test cases. See templates/mqtt-perf-test-case.yaml.
+
+# Pipeline requires custom agent pool (defined in $(pool.name)) 
+# with at least two separate agents:
+# - with 'mqtt-perf-tests-pub' demand - for publisher node
+# - with 'mqtt-perf-tests-sub' demand - for subscriber node
+# And two SSH service connections for AMD64 and ARM32 test hosts:
```

Comment:
SSH service connections, does it mean service connection defined in DevOps?

Reply:
Yes, service connection and private keys stored in dev ops, and not on the actual machines.

Reply:
how can this solution be scaled out? for example, I want to run master and RC branch for MQTT perf test.

Reply:
Due to the limited infrastructure, right now you can run only one test at a time. But you can pick any branch you want to test when you queue the new pipeline.

Reply:
Please think about how to support to run tests in parallel.  Probably in another PR for improvement.